src/App.vue

```diff
@@ -22,53 +37,116 @@ const transitionInterval = 0
 export default {
   name: 'app',
   created: function () {
+    this.newGame()
     window.addEventListener('keydown', this.move)
   },
   data: function () {
     return {
-      tiles: [
-               {x: 1, y: 2, number: 1},
-               {x: 2, y: 2, number: 2},
-               {x: 3, y: 2, number: 3}
-             ]
+      tiles: [],
+      score: 0
     }
   },
   methods: {
+    newGame() {
+      this.initializeTiles()
+      this.score = 0
+    },
+
+    initializeTiles() {
```

Comment:
This function is doing too much. By scanning this function I'm not able to quickly get an understanding of what its intent is. It would help if you were to break this function down to smaller functions, and name those functions appropriately so I can understand what those functions do.